README.md

```diff
@@ -59,17 +67,21 @@ import '~network-canvas-ui/styles/all.scss';
 
 - LIST TBC
 
-## Contributing
+## Contributing / Development
 
 ### Components
 
-- Add your components in `/src/`, link to them in the relevant `index.js`.
+- Run `npm run dev` to run a dev server and make chnages to components and styles in `/src/`, then link to them in the relevant `index.js`.
 - Run the babel build so that they can be imported
 
   ```sh
   $ yarn run prepublish
   ````
+  or
+  ```sh
+  $ npm run prepublish
+  ````
 
 ### Styles
 
-Just add the styles in `/styles/` and link to them in the relevant `_all.scss`, and you're done!
+Just add the styles in `/styles/` and link to them in the relevant `main.scss`, and you're done!
```

Comment:
I agree with Jamie. I think `main.scss` is a whole stylesheet for these purposes.

Is this ready to merge?